<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionWidget</class>
 <widget class="QWidget" name="ConnectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>441</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>118</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QCheckBox" name="autoconnect">
     <property name="text">
      <string>Automatically connect to this network when it is available</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="allUsers">
     <property name="text">
      <string>All users may connect to this network</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="KPushButton" name="pushButtonPermissions">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Edit advanced permissions for this connection</string>
     </property>
     <property name="whatsThis">
      <string>Fine-grained permissions editor for this connection. It lets you choose which users can activate/modify/delete this connection.</string>
     </property>
     <property name="text">
      <string>Advanced...</string>
     </property>
     <property name="icon">
      <iconset theme="preferences-desktop-user">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Firewall zone:</string>
     </property>
     <property name="buddy">
      <cstring>firewallZone</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QCheckBox" name="autoconnectVpn">
     <property name="text">
      <string>Automatically connect to VPN when using this connection</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="KComboBox" name="vpnCombobox">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="KComboBox" name="firewallZone">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAlphabetically</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>autoconnect</tabstop>
  <tabstop>allUsers</tabstop>
  <tabstop>pushButtonPermissions</tabstop>
  <tabstop>autoconnectVpn</tabstop>
  <tabstop>vpnCombobox</tabstop>
  <tabstop>firewallZone</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>allUsers</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButtonPermissions</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
